### ISSUE TITLE
Mav's latest version of DeX

### ISSUE THREAD

**Issue by hotelechoxrayindiasierra** [urothis/nwn-module-DungeonEternalX#8]
I feel that we should use the last iteration of DeX as a starting point for this community for a few reasons. 

It fixed a TON of issues that players constantly complained about, such as the bank broken, faction battlefield being a mess, fixes for SD that most of the player base enjoyed.

New dungeons were added to that version...

A few class changes and subraces were added that I can't remember off the top of my head. 

There's a myriad of things that the last version of DeX addressed, it took extensive effort and time to fix said issues. It would solve a lot of headaches to simply use that mod and build from there. Though at this point I feel it might be too little too late.

**Comment by Yazkir** [comment]
I agree with dungeons but not the weapons that were included with said dungeons. I think that PvE should reward varying levels of fame and maybe some less common magical items. Like, potions, scrolls, etc.

**Comment by hotelechoxrayindiasierra** [comment]
I have to admit I am pretty bias when it comes to gear... I like scripting it and making things unique. I tried very hard to make relatively mundane items for DeX that didn't impact overall balance. Though I did think it was pretty damn hilarious when people were running around with dual wielding bastard sword rangers orcs.

**Comment by DM-Chuck** [comment]
i agree using mavs latest mod would probably be the best course of action but i would argue against possibly not using the dungeons because of the lag issues that were tied to them. trying to hold a tourney or pvp in parries became very difficult when people were doing dungeon runs.

**Comment by TertiusL** [comment]
Do we have a list of all the changes on Mav's version? Then we could take a look at what we need vs what is going to cause issues. NWNEE cant really handle too many things going on at the same time, so if there are things thats going to make the lag worse... rather keep it out.

**Comment by milliorn** [comment]
It would be easier to note what changes where made and just import them.  Then you avoid having to remove resources which is more difficult task.

**Comment by Yazkir** [comment]
Yeah, I agree with Milliorn here. Importing is a much easier task. The current cleanup task is the number one priority though. I understand that people want things now, but patience is needed. The cleanup will make things much easier to build on top of afterwards.
